app/script/storage/StorageSchemata.js

```diff
@@ -310,6 +310,10 @@ z.storage.StorageSchemata = class StorageSchemata {
         },
         version: 15,
       },
+      {
+        schema: {},
```

Comment:
We need an empty change in the DB cause dexie-observable is creating some extra tables and indexes